anyblok_furetui/furetui/menus.py

```diff
@@ -124,7 +124,6 @@ def get_for(cls, parent, authenticated_userid):
         # TODO raise if MENU_TYPE is None
         query = cls.query()
         query = query.filter(cls.parent == parent)
-        query = query.order_by(cls.sequence)
```

Comment:
what is the pb, the order by is important here

Reply:
The order by is done twice. If you inspect the file you can find it on https://github.com/AnyBlok/anyblok_furetui/blob/92099e27f1a9a90fa089c0162f57cc5108f1c737/anyblok_furetui/furetui/menus.py#L130.
And this is a problem for MsSQL be cause it produces a query with `order by sequence, sequence`

Reply:
we must found why, and let the order by